go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/evergreen-ci/evergreen
 
-go 1.20
+go 1.19
```

Comment:
Was this change intended? :0

Reply:
No, I did that to make gqlgen work locally, thanks. Reverting